src/interactivity/onecore/ConIoSrvComm.cpp

```diff
@@ -410,16 +399,16 @@ VOID ConIoSrvComm::HandleFocusEvent(PCIS_EVENT Event)
                 // Relinquish control of the graphics device (only one
                 // DirectX application may control the device at any one
                 // time).
-                LOG_IF_FAILED(pWddmConEngine->Disable());
+                LOG_IF_FAILED(wddmConEngine.Disable());
 
                 // Let the Console IO Server that we have relinquished
                 // control of the display.
                 ReplyEvent.Type = CIS_EVENT_TYPE_FOCUS_ACK;
-                Ret = WriteFile(_pipeWriteHandle,
-                                &ReplyEvent,
-                                sizeof(CIS_EVENT),
-                                NULL,
-                                NULL);
+                WriteFile(_pipeWriteHandle,

```

Comment:
do we care that this might fail? the previous code did

Reply:
As discussed outside of GitHub: The previous code ignored the return value of `WriteFile`.